lib/scala/pkg/src/main/scala/org/enso/pkg/Script.scala

```diff
@@ -0,0 +1,80 @@
+package org.enso.pkg
+
+import io.circe.{Decoder, DecodingFailure, Encoder, Json}
+import org.enso.scala.yaml.{YamlDecoder, YamlEncoder}
+import org.yaml.snakeyaml.error.YAMLException
+import org.yaml.snakeyaml.nodes.{MappingNode, Node, ScalarNode, SequenceNode}
+
+import java.util
+
+case class Script(name: String, arguments: Seq[String])
+
+object Script {
+
+  val knownScripts = List("refresh")
+
+  /** [[Encoder]] instance for the [[Script]]. */
+  implicit val encoder: Encoder[Script] = { script =>
+    val vs = script.arguments.map(Json.fromString)
+    Json.obj(script.name -> Json.arr(vs: _*))
+  }
+
+  implicit val yamlEncoder: YamlEncoder[Script] =
+    new YamlEncoder[Script] {
+      override def encode(value: Script) = {
+        val fields = new util.ArrayList[String](value.arguments.length)
+        value.arguments.foreach(v => fields.add(v))
+        toMap(value.name, fields)
+      }
+    }
+
+  private def find(
+    name: String,
+    values: Seq[String]
+  ): Either[String, Script] = {
+    if (knownScripts.contains(name)) Right(Script(name, values))
+    else Left("Uknown script: " + name)
+  }
+
+  /** [[Decoder]] instance for the [[Script]]. */
+  implicit val decoder: Decoder[Script] = { json =>
+    for {
+      key    <- json.key.toRight(DecodingFailure("no key", Nil))
+      fields <- json.get[List[String]](key)
+      script <- find(key, fields).left.map(v => DecodingFailure(v, Nil))
```

Comment:
We should not fail the whole `scripts` section if one element fails. It is bad for backward compatibility. In general, we should allow arbitrary config structure and ignore everything irrelevant.

Reply:
This is a bit random as we sometimes fail and sometimes not, when parsing Config. But if I remove the known scripts check then that will resolve itself.